WebApiCoreSeed.Domain/WebApiCoreSeed.Domain.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="1.1.2" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="1.1.1" />

```

Comment:
Minor: No versioning issues? Should we consider going all 1.1.1 or 1.1.2?

Reply:
I'll use 1.1.2 for all, just in case